contracts/ETO/ETOTokenTerms.sol

```diff
@@ -85,14 +88,19 @@ contract ETOTokenTerms is Math, IContractId {
         EQUITY_TOKEN_NAME = equityTokenName;
         EQUITY_TOKEN_SYMBOL = equityTokenSymbol;
         EQUITY_TOKENS_PER_SHARE = equityTokensPerShare;
+        EQUITY_TOKENS_PRECISION = equityTokensPrecision;
+        EQUITY_TOKENS_POWER = 10**uint256(EQUITY_TOKENS_PRECISION);
+
```

Comment:
> require(equityTokensPerShare > 0);

to avoid weird situations where 1 token is more than 1 share we could 

 ```require(equityTokensPerShare > EQUITY_TOKENS_POWER);